### DISCUSSION TITLE
CollectionView seems to require ScrollView in .NET MAUI

### DISCUSSION THREAD

**Discussion by imsam67** [dotnet/maui#8095] [Q&A]
Looks like we now need to wrap `CollectionView` in `ScrollView` in .NET MAUI. If I'm not mistaken, in Xamarin Forms, we were told not to do this.

Am I right that we now need to wrap `CollectionView` in a `ScrollView`?

P.S. Here's a sample I did. Without `ScrollView`, the list is stuck and I can't see the bottom entries because I can't scroll. Once I add `ScrollView`, it works fine.

[https://github.com/imsam67/maui-collectionview](https://github.com/imsam67/maui-collectionview)

**Comment by hartez** [comment]
A ScrollView should _not_ be required when using a CollectionView - the behavior you're seeing is likely a bug. Please open an issue.
